docs/hydra/sdk/go.md

```diff
@@ -9,62 +9,88 @@ To install the Go SDK, run:
 go get -u -d github.com/ory/hydra-client-go
 ```
 
+:::info
+
+Be aware that endpoints `/oauth2/auth` and `/oauth2/token` **MUST NOT** be consumed
+using this SDK. Use [golang.org/x/oauth2](https://godoc.org/golang.org/x/oauth2)
+instead.
```

Comment:
Can you give me the reason why? It'd be good to include it in the note. 
Also, there are multiple "note" formats supported by default. For a message like this one you can use
```
:::warning
```

Reply:
@vinckr you can also use suggestions inline:

````
```suggestion
Be aware that endpoints `/oauth2/auth` and `/oauth2/token` **MUST NOT** be consumed
using this SDK. Use [golang.org/x/oauth2](https://godoc.org/golang.org/x/oauth2)
instead.
```
````